lib/enkf/enkf_main.cpp

```diff
@@ -70,6 +70,7 @@
 #include <ert/res_util/res_util_defaults.hpp>
 #include <ert/res_util/subst_func.hpp>
 #include <ert/res_util/res_log.hpp>
+#include <ert/res_util/es_testdata.hpp>
```

Comment:
Are there any programmatic changes to this file? If not, I suggest to remove the newly added scope to keep the changes small... Please correct me if I'm wrong!

The improved comments are of course very much welcome 👍

Reply:
> I suggest to remove the newly added scope to keep the changes small.

Everyone complain about my scopes; when I become dictator then there will be scopes - yessir; but for now I have removed it :-)

Reply:
Scopes went out of fashion in C99... ;)